hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -76,10 +76,10 @@
   private static final long serialVersionUID = 1L;
   private static final Logger LOG = LogManager.getLogger(HoodieTableMetaClient.class);
   public static final String METAFOLDER_NAME = ".hoodie";
-  public static final String TEMPFOLDER_NAME = METAFOLDER_NAME + File.separator + ".temp";
-  public static final String AUXILIARYFOLDER_NAME = METAFOLDER_NAME + File.separator + ".aux";
-  public static final String BOOTSTRAP_INDEX_ROOT_FOLDER_PATH = AUXILIARYFOLDER_NAME + File.separator + ".bootstrap";
-  public static final String HEARTBEAT_FOLDER_NAME = METAFOLDER_NAME + File.separator + ".heartbeat";
+  public static final String TEMPFOLDER_NAME = METAFOLDER_NAME + Path.SEPARATOR + ".temp";
```

Comment:
could you please handle `File.separator` usage in other place as well?

Reply:
Hello, may I ask whether it needs to be dealt with elsewhere or this problem needs to be dealt with elsewhere

Reply:
yes, i would prefer to handle it as well in other places.

Reply:
OK, it is possible I think.

Reply:
I have also modified and tested it in other places. Hudi's performance is normal. I hope you can review the code when you have time。

Reply:
@Carl-Zhou-CN I still see there are still some `File.separator` usage(you can search File.separator globally in IntelliJ IDEA ), would you please handle them as well?

Reply:
@leesf OK, thank you for your suggestion. I have modified everything, including test cases. I hope you can help me review the code